client/src/pages/events/components/constants.js

```diff
@@ -1,6 +1,14 @@
-import { EVENTS_22_23 } from 'assets/index';
+import { EVENTS_22_23, EVENTS_23_24 } from 'assets/index';
 
 const PAST_EVENTS = {
+	'2023/2024': [
+		{
+			image: EVENTS_23_24[0],
+			title: 'Summer Hiring',
+			date: 'Thursday, June 6 | 11.59PM',
```

Comment:
Can you change `.` to `:` for the time? Apply this to other events as well:
<img width="848" alt="image" src="https://github.com/gisauweb/GISAUBC/assets/65900990/3d7932c2-9fe2-4f42-8451-8b00bd9a06f8">